### ISSUE TITLE
Feature request: mark code examples as false positive for future debug (and reporting)

### ISSUE THREAD

**Issue by biow0lf** [rubocop-hq/rubocop#4594]
In some cases I should like to mark code line as false positive and make rubocop ignore for now.

E.g. `# rubocop:falsepositive CopName`. We can use this information later for debug. Regular, I don't have enough time to write good bug report. Probably, I am not alone.

Then, for example, in the end of the week, update rubocop from git, run and fix and/or report.

## Expected behavior

Left blank.

## Actual behavior

Left blank.

## Steps to reproduce the problem

Left blank.

## RuboCop version

```
$ bundle exec rubocop -V
0.49.1 (using Parser 2.4.0.0, running on ruby 2.4.1 x86_64-darwin16)
```

**Comment by ryanhageman** [comment]
This would be an interesting feature.

Would using `OPTIMIZE` or `TODO` notes work for this as a workaround?

Use `# rubocop:disable CopName` and then on the next line `# OPTIMIZE rubocop:falsepositive CopName`.

When you're ready, you can run `rake notes` and find all of your comments:

```shell
~> rake notes
app/controllers/application_controller.rb:
  * [7] [OPTIMIZE] rubocop:falsepositive CopName
```
